src/Microsoft.DotNet.Build.Tasks/PackageFiles/PerfTesting.targets

```diff
@@ -1,32 +1,31 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="12.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
 
-    <!-- Perf Runner NuGet package paths -->
+  <!-- Perf Runner NuGet package paths -->
+  <PropertyGroup>
+   <XunitPerfAnalysisPackageVersion Condition="'$(XunitPerfAnalysisPackageVersion)' == ''">1.0.0-alpha-build0037</XunitPerfAnalysisPackageVersion>
```

Comment:
The versions have to be kept in sync as of now (with in-sync changes to build-tools plus all related products) which is what I am trying to do away with, don't mind throwing an error message if the value is not found if that makes more sense